tests/frontend/test-class-wpseo-frontend.php

```diff
@@ -151,6 +151,7 @@ public function test_webmaster_tools_authentication_home() {
 
 		$this->go_to_home();
 
+		$this->run_webmaster_tools_authentication_option_test( 'baiduverify', 'h95vqTAZZq', '<meta name="baidu-site-verification" content="h95vqTAZZq" />' . "\n" );
```

Comment:
This looks like the actual verification key. Could you please replace this with something else?